201-vm-linux-jupyterhub/azuredeploy.json

```diff
@@ -312,7 +326,24 @@
                         "offer": "UbuntuServer",
                         "sku": "18.04-LTS",
                         "version": "latest"
-                    }
+                    },
+                    "copy": [
```

Comment:
remove the copy loop, you're only creating one disk

Reply:
bringing alignment to 101-vm-linux-jupyerhub template but having larger machine sizes